modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -628,18 +640,62 @@
           it "returns a claim with the 'claimId' and 'lighthouseId' set" do
             with_okta_user(scopes) do |auth_header|
               VCR.use_cassette('bgs/tracked_items/find_tracked_items') do
-                expect_any_instance_of(BGS::EbenefitsBenefitClaimsStatus)
-                  .to receive(:find_benefit_claim_details_by_benefit_claim_id).and_return(bgs_claim)
-                expect(ClaimsApi::AutoEstablishedClaim)
-                  .to receive(:get_by_id_and_icn).and_return(nil)
+                VCR.use_cassette('evss/documents/get_claim_documents') do
+                  expect_any_instance_of(BGS::EbenefitsBenefitClaimsStatus)
+                    .to receive(:find_benefit_claim_details_by_benefit_claim_id).and_return(bgs_claim)
+                  expect(ClaimsApi::AutoEstablishedClaim)
+                    .to receive(:get_by_id_and_icn).and_return(nil)
 
-                get claim_by_id_path, headers: auth_header
+                  get claim_by_id_path, headers: auth_header
 
-                json_response = JSON.parse(response.body)
-                expect(response.status).to eq(200)
-                expect(json_response).to be_an_instance_of(Hash)
-                expect(json_response['claimType']).to eq('value from BGS')
-                expect(json_response['status']).to eq('PENDING')
+                  json_response = JSON.parse(response.body)
+                  expect(response.status).to eq(200)
+                  expect(json_response).to be_an_instance_of(Hash)
+                  expect(json_response['claimType']).to eq('value from BGS')
+                  expect(json_response['status']).to eq('PENDING')
+                end
+              end
+            end
+          end
+        end
+      end
+
+      describe "handling the 'supporting_documents'" do
+        context 'it has documents' do
```

Comment:
Can we add specs for these situations?
- when the claim has no docs, we make sure that `supportingDocuments` is an empty array
- when the BGS-version of a claim doesn't exist, we make sure that `supportingDocuments` is an empty array